### ISSUE TITLE
Clear cache from Admin UI throws exception

### ISSUE THREAD

**Issue by mugge6** [pimcore/pimcore#9575]
### Expected behavior
Clearing all caches from Admin UI should work without any exception.

### Actual behavior

Cannot clear all caches from Admin UI after Pimcore X upgrade. 

Configured cache backend is a Redis instance:

```
framework:
    cache:
        pools:
            pimcore.cache.pool:
                public: true
                tags: true
                default_lifetime: 31536000  # 1 year
                adapter: pimcore.cache.adapter.redis_tag_aware
                provider: 'redis://redis'
```

Clear cache from Admin UI:

![image](https://user-images.githubusercontent.com/16953779/123407183-f61aaf80-d5ab-11eb-9626-f49467b32d66.png)

The following error details will be displayed:

```
Timestamp: Fri Jun 25 2021 11:27:01 GMT+0200 (Central European Summer Time)
Status: 500 | Internal Server Error
URL: /admin/settings/clear-cache
Method: DELETE
Message: An exception occurred while executing 'truncate table cache_items':

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'fumo.cache_items' doesn't exist
Trace: 
in /var/www/pimcore/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:61
#0 /var/www/pimcore/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php(182): Doctrine\DBAL\Driver\AbstractMySQLDriver->convertException('An exception oc...', Object(Doctrine\DBAL\Driver\PDO\Exception))
#1 /var/www/pimcore/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php(159): Doctrine\DBAL\DBALException::wrapException(Object(Doctrine\DBAL\Driver\PDO\MySQL\Driver), Object(Doctrine\DBAL\Driver\PDO\Exception), 'An exception oc...')
#2 /var/www/pimcore/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(2212): Doctrine\DBAL\DBALException::driverExceptionDuringQuery(Object(Doctrine\DBAL\Driver\PDO\MySQL\Driver), Object(Doctrine\DBAL\Driver\PDO\Exception), 'truncate table ...', Array)
#3 /var/www/pimcore/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(1299): Doctrine\DBAL\Connection->handleExceptionDuringQuery(Object(Doctrine\DBAL\Driver\PDO\Exception), 'truncate table ...', Array, Array)
#4 /var/www/pimcore/vendor/pimcore/pimcore/lib/Db/PimcoreExtensionsTrait.php(76): Doctrine\DBAL\Connection->executeQuery('truncate table ...')
#5 /var/www/pimcore/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/SettingsController.php(623): Pimcore\Db\Connection->query('truncate table ...')
#6 /var/www/pimcore/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(157): Pimcore\Bundle\AdminBundle\Controller\Admin\SettingsController->clearCacheAction(Object(Symfony\Component\HttpFoundation\Request), Object(App\Kernel), Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher), Object(Pimcore\Cache\Core\CoreCacheHandler), Object(Pimcore\Db\Connection), Object(Symfony\Component\Filesystem\Filesystem), Object(Pimcore\Cache\Symfony\CacheClearer))
#7 /var/www/pimcore/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(79): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#8 /var/www/pimcore/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(199): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#9 /var/www/pimcore/public/index.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#10 {main}
```

Additional remark:
After upgrading to Pimcore X there is no "cache*" table at all in the database.

**Comment by brusch** [comment]
Fixed by #9578

**Comment by Yivan** [comment]
@brusch Thanks for the fix.
Just to mention this occurs when upgrading from Pimcore 6 to 10 too, when executing migrations part:

![image](https://user-images.githubusercontent.com/9025839/124954734-b1b0fa00-e016-11eb-8f10-aa2a6bdc12df.png)

Does this problem impact upgrade or those messages can be safely ignored ? (maybe a note on this could be added on upgrade doc page to say it is not a problem, waiting 10.1 becomes the stable release ?)
